src/components/common/AnimationControls.js

```diff
@@ -26,10 +28,18 @@ import {
 } from '../../config/constants';
 
 const useStyles = makeStyles(() => ({
+  buttonContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  button: {
+    fontSize: '1.75em',
+  },
   pauseButton: {
     color: yellow[800],
   },
   playButton: { color: green[800] },
+  stopButton: { color: blue[900] },
   resetButton: { color: red[800] },
```

Comment:
The colors are different from the 2 other apps. Maybe let's go for 
pause => yellowish-orange
reset => orange
stop => red
See the second app here https://apps.dev.graasp.eu/5acb589d0d5d9464081c2d46/602bdf211db0d51cb392aeb9/latest/index.html

Reply:
👍 good catch